### PR TITLE
ci: Add a script that verifies Appium deployment on different OSs

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        # node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install Appium
+        run: npm i -g appium@next
```
Comment (jlipps) [inline .github/workflows/deploy.yml:28]: what if this script runs on a git tag `appium@2.3.1`? the npm `next` tag might still point to something else? do we not want to run `npm i -g appium@${{ tag }}` assuming we can get the tag as a variable?

**Review by jlipps — APPROVED** [review]
very cool, so this is like a postpublish script that we run whenever we make a `next` release?

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (mykola-mokhnach) [reply .github/workflows/deploy.yml:28]: `tag` is only defined when the pipeline is triggered by a new tag. It is going to be unset if the pipeline is triggered manually though

Comment (mykola-mokhnach) [reply .github/workflows/deploy.yml]: changed to use the current tag

**Pull request merged**